databroker/broker.py

```diff
@@ -433,7 +433,12 @@ def add_filter(self, **kwargs):
         :meth:`Broker.clear_filters`
 
         """
-        self.filters.append(dict(**kwargs))
+        d = dict(**kwargs)
+        for val in self.filters:
+            for k,v in val.items():
+                if k in d.keys():
+                    raise ValueError('Filter key already exists.')
```

Comment:
Can you include in the error message what the filter it conflicts with is and how to fix the problem?